### PR TITLE
social buttons w/ follow functionality

### LINKED ISSUE #438: [BUTTON IDEA]: Follow Social Buttons
**1. Name of button**: Follow Social Button

**2. Description**: A social button to follow the creators social media handles

**3. Button Type (Animated, Special, etc...)**: Social

**4. Will you work on it?**: no

**submitted by**: @Archis19

### PR DESCRIPTION
**Pull request by adsingh14**
Added three social icons.

Issue:  No

### CODE REVIEW

File: src/components/social/_snapchat.less [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.social-btn {
+  &.snapchat-ghost {
```
Comment (shahednasser) [inline src/components/social/_snapchat.less:2]: can you remove the ghost in the class name just for naming conventions

**Review by shahednasser — CHANGES_REQUESTED** [review]
It all looks good! I just have one simple comment below. Please run `npm run compile` after you make the changes.

File: src/components/social/_snapchat.less (hunk 1 above)
Comment (adsingh14) [reply src/components/social/_snapchat.less:2]: Alright !

**Comment by adsingh14** [comment]
> 
> It all looks good! I just have one simple comment below. Please run `npm run compile` after you make the changes.

Maybe, I forgot that ! Revisions will be done in a couple of minutes. 🚀

File: src/components/social/_snapchat.less (hunk 1 above)
Comment (adsingh14) [reply src/components/social/_snapchat.less:2]: Alright !

**Review by adsingh14 — COMMENTED** [review]
Fixed !

**Comment by shahednasser** [comment]
Thank you for the great work!

@all-contributors please add @adsingh14 for code

**Pull request merged**